cilium/cmd/helpers.go

```diff
@@ -185,3 +189,84 @@ func expandNestedJSON(result bytes.Buffer) (bytes.Buffer, error) {
 
 	return result, nil
 }
+
+// parseTrafficString converts the provided string to its corresponding
+// TrafficDirection. If the string does not correspond to a valid TrafficDirection
+// type, returns Invalid and a corresponding error.
+func parseTrafficString(td string) (policymap.TrafficDirection, error) {
+	lowered := strings.ToLower(td)
+	if lowered == "ingress" {
+		return policymap.Ingress, nil
+	} else if lowered == "egress" {
+		return policymap.Egress, nil
+	} else {
+		return policymap.Invalid, fmt.Errorf("invalid direction %q provided", td)
+	}
+
+}
+
+func updatePolicyKey(cmd *cobra.Command, args []string, add bool) {
```

Comment:
Since this is now in a common place, could you add a comment explaining what it does. Separately, it's a little weird that we take a cobra.Command and args directly here. The arg length check is more relevant to where we implement the top-level cli command logic. I suggest factoring out the args usage and giving the function more explicit parameters.